src/benchmark/trulens/benchmark/benchmark_frameworks/experiments/competitive_analysis.ipynb

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">do you need any sampling ?
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p><code>qags_xsum_true_labels</code> is derived from <code>qags_xsum</code>,where the sampling already happened     </p>